packages/angular-output-target/src/generate-value-accessors.ts

```diff
@@ -54,30 +55,34 @@ export default async function generateValueAccessors(
       const srcFilePath = path.join(__dirname, '../resources/control-value-accessors/', targetFileName);
       const srcFileContents = await compilerCtx.fs.readFile(srcFilePath);
 
-      const finalText = createValueAccessor(srcFileContents, normalizedValueAccessors[valueAccessorType]);
+      const finalText = createValueAccessor(
+        srcFileContents,
+        normalizedValueAccessors[valueAccessorType],
+        outputTarget.outputType
+      );
       await compilerCtx.fs.writeFile(targetFilePath, finalText);
     })
   );
 
   await copyResources(config, ['value-accessor.ts'], targetDir);
 }
 
-export function createValueAccessor(srcFileContents: string, valueAccessor: ValueAccessor) {
+export function createValueAccessor(srcFileContents: string, valueAccessor: ValueAccessor, outputType?: OutputType) {
   const hostContents = valueAccessor.eventTargets.map((listItem) =>
     VALUE_ACCESSOR_EVENTTARGETS.replace(VALUE_ACCESSOR_EVENT, listItem[0]).replace(
       VALUE_ACCESSOR_TARGETATTR,
       listItem[1]
     )
   );
-
   return srcFileContents
     .replace(VALUE_ACCESSOR_SELECTORS, valueAccessor.elementSelectors.join(', '))
-    .replace(VALUE_ACCESSOR_EVENTTARGETS, hostContents.join(`,${EOL}`));
+    .replace(VALUE_ACCESSOR_EVENTTARGETS, hostContents.join(`,${EOL}`))
+    .replace(VALUE_ACCESSOR_STANDALONE, outputType && outputType !== OutputTypes.Component ? ',standalone: true' : '');
```

Comment:
I think we need to update this check so it _only_ applies standalone for `OutputTypes.Standalone`, otherwise it'll set standalone to `true` if they supply `OutputTypes.Scam`, which I don't think is appropriate.

```suggestion
    .replace(VALUE_ACCESSOR_STANDALONE, outputType && outputType === OutputTypes.Standalone ? ',standalone: true' : '');
```

Would be good to add a test for this as well

Reply:
Wll do!